jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/objectActions/ViewObjectVersionAction.kt

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.services.s3.objectActions
+
+import com.intellij.icons.AllIcons
+import software.aws.toolkits.jetbrains.services.s3.editor.S3TreeNode
+import software.aws.toolkits.jetbrains.services.s3.editor.S3TreeObjectNode
+import software.aws.toolkits.jetbrains.services.s3.editor.S3TreeTable
+import software.aws.toolkits.resources.message
+
+class ViewObjectVersionAction constructor(treeTable: S3TreeTable) :
+    SingleS3ObjectAction(treeTable, message("s3.version.history.view"), AllIcons.Actions.Find) {
```

Comment:
I do not see a good icon for this, but `AllIcons.Actions.ShowAsTree` may be better. Find indicates search IMO.

Reply:
Look better :+!: